BoxContentSDK/BoxContentSDK/OAuth2/BOXAbstractSession.m

```diff
@@ -258,8 +258,8 @@ + (NSArray *)usersInKeychain
     }
     
     NSArray *sortedUsers = [users sortedArrayUsingComparator:^NSComparisonResult(id a, id b) {
-        BOXUserMini *userA = (BOXUserMini *) a;
-        BOXUserMini *userB = (BOXUserMini *) b;
+        id<UniqueSDKUser> userA = (id<UniqueSDKUser>) a;
```

Comment:
You can declare `a` and `b` as `id<UniqueSDKUser>` in the block parameters, and you don't need the casts.

Reply:
I wonder if this sort is even necessary, but we don't have to address that here.